basics/Main.java

```diff
@@ -1,5 +1,73 @@
+import java.time.LocalDateTime;
+
 public class Main {
   public static void main(String[] args) {
-    System.out.println("Hello World!");
+    // System.out.println("Hello World!");
+
+    int dogCount = 1;
+    System.out.println("I own " + dogCount + " " + pluralize("dog", dogCount) + ".");
+    int catCount = 2;
+    System.out.println("I own " + catCount + " " + pluralize("cat", catCount) + ".");
+    int turtleCount = 0;
+    System.out.println("I own " + turtleCount + " " + pluralize("turtle", turtleCount) + ".");
+
+    flipNHeads(1);
+    // flipNHeads(2);
+
+    Main object = new Main();
+    object.clock();
   }
+
+  // Write a function called pluralize that accepts a word and a number and returns a string with the word pluralized with an “s” if the number is zero, or greater than one.
+  public static String pluralize(String animal, int n) {
+    char s = 's';
+    if (n == 1) {
+      return animal;
+    } else if (n > 0) {
+      return animal + s;
+    }
+    else {
+      return animal + s;
+    }
+  }
+
+  // Write a function called flipNHeads that accepts an integer n and flips coins until n heads are flipped in a row. Simulate coin flipping by choosing a random number between 0 and 1. Numbers below .5 are considered tails. Numbers at and above .5 are considered heads. Print out heads or tails on one line for each flip. Print It took FLIPS flips to flip N heads in a row. once n heads have been in a row.
+  public static void flipNHeads(int n) {
+    double rand = Math.random();
+    int flipCounter = 0;
+    int headsCounter = n;
+
+    while (headsCounter != 0) {
+      if (rand > 0.5) {
+        System.out.println(String.format("heads"));
+        flipCounter++;
+        headsCounter--;
+      } else {
+        System.out.println(String.format("tails"));
```

Comment:
Else if you get tails, you should zero out the heads count, `headsCount = 0;` Since we are tracking how many heads in a row.